service/storage_service.cc

```diff
@@ -2742,6 +2766,10 @@ std::unordered_multimap<dht::token_range, inet_address> storage_service::get_cha
 
 // Runs inside seastar::async context
 void storage_service::unbootstrap() {
+    db::get_local_batchlog_manager().do_batch_log_replay().get();
+    decommission_with_repair(_db, _token_metadata).get();
+    leave_ring();
+#if 0
```

Comment:
Please, remove all sections that you put under #if 0 it would make it easier to review (we'd see all lines that are removed) and would leave the code leaner after the patch is merged.

Reply:
I am not in a rush to remove the streaming based method. We probably need a option to fallback to the old method in case the method does not work in the field. I am actually using the #if 0 to reduce the number of changes in this patch set.